.changes/v3.5.0/762-improvements.md

```diff
@@ -0,0 +1 @@
+* **Resource** and **Data source** `vcd_vapp` have a new `lease` block to view and change lease settings.
```

Comment:
```suggestion
* **Resource** and **Data source** `vcd_vapp` a new `lease` block to view and change lease settings.
```

Reply:
Why removing the verb? Now the object ("a new lease block") is hanging.

Reply:
True, better transform it to the format used previously with "adds" word, few examples:

"datasource/vcd_resource_list adds vcd_right, vcd_role, vcd_global_role, vcd_rights_bundle to the supported resource types (#677)"
"resource/vcd_vapp_vm adds support to update storage_profile (#580)"

Reply:
Updated